Dockerfile

```diff
@@ -2,6 +2,8 @@ FROM python:2.7
 
 COPY . /securitybot
 
+USER securitybot
```

Comment:
I believe this is going to throw an error, since the user will not exist.

Adding a new user up above is the way to go, something like this:
`RUN adduser --disabled-password --gecos '' securitybot`